dashboard/lib/policies/lti.rb

```diff
@@ -219,4 +223,8 @@ def self.feedback_available?(user)
   def self.lti_registration_in_progress?(session)
     PartialRegistration.in_progress?(session) && PartialRegistration.get_provider(session) == AuthenticationOption::LTI_V1
   end
+
+  def self.account_linking?(session, user)
+    session[:lms_landing].present? && Policies::Lti.only_lti_auth?(user) && !user.lms_landing_opted_out
```

Comment:
Since this is referencing another class method in the same class, I think it can just be `only_lti_auth?` without specifying Policies::Lti

Reply:
Good catch!